content/healthchecks/livenessprobe.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Configure Liveness Probe"
+chapter: true
+weight: 5
+---
+
+# Configure Liveness Probe
+
+```
```

Comment:
Instead of downloading liveness-app.yaml on the intro, let's show the code here and have the user save the file as `~/environment/healthchecks/liveness-app.yaml`. Explain/highlight the _livenessProbe_ settings and what we expect to happen.